migrator/src/main/scala/com/scylladb/migrator/writers/DynamoDB.scala

```diff
@@ -9,14 +9,69 @@ import org.apache.hadoop.mapred.JobConf
 import org.apache.log4j.LogManager
 import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.SparkSession
-import software.amazon.awssdk.services.dynamodb.model.{ AttributeValue, TableDescription }
+import software.amazon.awssdk.services.dynamodb.model.{
+  AttributeValue,
+  DeleteItemRequest,
+  TableDescription
+}
 
 import java.util
+import java.util.stream.Collectors
 
 object DynamoDB {
 
   val log = LogManager.getLogger("com.scylladb.migrator.writers.DynamoDB")
 
+  def deleteRDD(target: TargetSettings.DynamoDB,
```

Comment:
Is there should be function for delete operations?

Reply:
I was told that current version will remove rows that should be deleted but keeps row with the "_dynamo_op_type"
which is what confuses me and I didn't try myself if it really is like that

Reply:
it is possible , since rec. event type is not used later and dynamowriteable item doesn't seem to have an optype

I am curious if the rdd we have could be reused for both operations to avoid creating new client connection